src/components/layouts/FocussedLayout.vue

```diff
@@ -2,20 +2,23 @@
 import useAlerts from '@/composables/useAlerts';
 import { useReturnRoute } from '@/composables/useReturnRoute';
 import AppNavAlert from '@/components/navs/AppNav/AppNavAlert.vue';
+import { isFakeModal } from '@/composables/useFakeModal';
 
 const { getReturnRoute } = useReturnRoute();
 const { currentAlert } = useAlerts();
 </script>
 <template>
   <AppNavAlert v-if="currentAlert" :alert="currentAlert" />
   <div class="pb-16">
-    <div class="mb-12 layout-header">
-      <div />
-      <BalBtn tag="router-link" :to="getReturnRoute()" color="white" circle>
-        <BalIcon name="x" size="lg" />
-      </BalBtn>
+    <div class="h-screen" :class="{ 'bg-gray-850': isFakeModal }">
```

Comment:
This is a hack to be able to use a page with modal style, allowing vertical scroll.